db/migrations/sdp-migrations/2024-08-20.0-alter-receiver-verifications-table-add-verification-channel.sql

```diff
@@ -0,0 +1,12 @@
+-- +migrate Up
+ALTER TABLE receiver_verifications
+    ADD COLUMN verification_channel message_channel;
+
+UPDATE receiver_verifications rv
+    SET verification_channel = 'SMS'::message_channel
+    WHERE rv.verification_channel IS NULL;
+
```

Comment:
👋  This update should only be applied to the entries where `confirmed_at IS NOT NULL`.

Reply:
Addressed in 837298c0b5cc13e50c7b802bfab3a5f20a63dd11